ingestion/src/metadata/ingestion/bulksink/metadata_usage.py

```diff
@@ -82,25 +83,20 @@ def write_records(self) -> None:
                 table_usage.database, table_usage.table
             )
             if table_entity is not None:
-                table_usage_request = TableUsageRequest(
-                    date=table_usage.date, count=table_usage.count
-                )
-                try:
-                    self.metadata.publish_table_usage(table_entity, table_usage_request)
```

Comment:
When there are two queries referring to same table but with different format, for example
`select * from test_dataset.session_log2 limit 50` & `select * from abiding-honor-344406.test_dataset.session_log2 limit 100;` 

Table usage creates a separate record for both of them but in metadata_usage both since both are referring to same table entity,
one table record gets published with count but the other record gets failed with message `Failed to update usage for session_log2 Entity already exists`

this also resulted in incorrect count of queries.